app/views/idv/how_to_verify/show.html.erb

```diff
@@ -11,57 +11,69 @@
+            <div classe="tablet:grid-col">
+              <%= image_tag asset_url('idv/remote.svg'),
+                            width: 88,
+                            height: 88,
+                            class: 'margin-right-1',
+                            alt: t('image_description.laptop_and_phone')
+              %>
+            </div>
+            <div class="tablet:grid-col">
+              <h2 class="h3 margin-top-0"><%= t('doc_auth.headings.verify_online') %></h2>
+              <div>
+                <p><%= t('doc_auth.info.verify_online_instruction') %></p>
+                <p><%= t('doc_auth.info.verify_online_description') %></p>
+              </div>
+              <%= f.submit t('forms.buttons.continue_remote'), class: 'display-block margin-y-5' %>
+            </div>
           </div>
         <% end %>
-    <%= f.radio_button(
+  </div>
+<% end %>
+<%= simple_form_for(
+      @idv_how_to_verify_form,
+      html: { autocomplete: 'off' },
+      method: :put,
+      url: idv_how_to_verify_url,
+    ) do |f|
+%>
+  <div role="group" aria-labelledby="how-to-verify-info">
+    <%= f.hidden_field(
           :selection,
-          Idv::HowToVerifyForm::IPP,
-          class: 'usa-radio__input usa-radio__input--tile',
+          value: Idv::HowToVerifyForm::IPP,
         ) %>
     <%= f.label(
           :selection_ipp,
-          class: 'usa-radio__label usa-radio__label--illustrated usa-radio__label-illustrated-large',
         ) do %>
-          <%= image_tag asset_url('idv/in-person.svg'),
-                        width: 88,
-                        height: 92,
-                        class: 'usa-radio__image',
-                        alt: t('image_description.post_office')
-          %>
-          <div class="usa-radio__label--text">
-            <h3><%= t('doc_auth.headings.verify_at_post_office') %></h3>
-            <span class="usa-radio__label-description">
-              <p><%= t('doc_auth.info.verify_at_post_office_instruction') %></p>
-              <p><%= t('doc_auth.info.verify_at_post_office_description') %></p>
-            </span>
+          <div class="grid-row">
+            <div classe="tablet:grid-col">
+              <%= image_tag asset_url('idv/in-person.svg'),
+                            width: 88,
+                            height: 88,
+                            class: 'margin-right-1',
+                            alt: t('image_description.post_office')
+              %>
+            </div>
+            <div class="tablet:grid-col">
+              <h3 class="margin-top-0"><%= t('doc_auth.headings.verify_at_post_office') %></h3>
+              <div>
+                <p><%= t('doc_auth.info.verify_at_post_office_instruction') %></p>
+                <p><%= t('doc_auth.info.verify_at_post_office_description') %></p>
+              </div>
+              <%= f.submit t('forms.buttons.continue_ipp'), class: 'display-block margin-y-5', outline: true %>
```

Comment:
Do we want the submit button inside of the label element? I don't think you want interactive elements inside of a label? [MDN Link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#accessibility_concerns)

Reply:
thanks for linking this documentation. I'm moving it into a different div, now just trying to figure out how to get the button to align with the block of text above it

Reply:
ok i was able to move the button so it aligns with the block of text...but it's not a super neat solution